ansible_builder/cli.py

```diff
@@ -143,6 +143,16 @@ def add_container_options(parser):
                                 ' and '.join([' for '.join([v, k]) for k, v in constants.runtime_files.items()]))
                        )
 
+        p.add_argument('--galaxy-keyring',
+                       help='Keyring for collection signature verification during installs from Galaxy. '
+                            'Will be copied into images. Verification is disabled if unset.')
+        p.add_argument('--galaxy-ignore-signature-status-code',
```

Comment:
This is a multi-value `type='append'` option in the Galaxy CLI, so this one probably needs to be as well- then we'll need to generate one `--ignore-signature-status-code` for each value we got. Could probably also update the description to match Galaxy CLI re: specifying multiple values.